Orchestration and Scheduling are Declarative (#2390)
dbt-labs/docs.getdbt.com: website/docs/community/resources/viewpoint.md

- code is deployed.

Workflows like this should be built to execute with a single command.

> ## The orchestration and control flow of analytics pipelines are configured declaratively 

Reviewer: ```suggestion
## Configure the orchestration and control flow of analytics pipelines declaratively 
```

> A data professional should maximize the time spent working with data. The configuration of where, how frequently, and in what order the transformations are executed should therefore be declartive, in order to minimize time operationalizing and maintaining workflows.

Reviewer: I'm not sure I'm capturing the intent appropriately so definitely open to changes, but here's my suggestion to make this sentence active:
```suggestion
A data professional should maximize their time spent working with data by minimizing time operationalizing and maintaining workflows. With this efficiency in mind, you should declaratively configure where, how frequently, and in what order you execute transformations.
```

> Pipelines should be deterministic, and not execute different set of transformstions based upsteam conditionals.

Reviewer: ```suggestion
Pipelines should be deterministic, and not execute different set of transformations based upstream conditionals.
```
